src/sagas/workspaces.ts

```diff
@@ -62,8 +70,12 @@ export default function* workspaceSaga(): SagaIterator {
     const globals: Array<[string, any]> = yield select(
       (state: IState) => (state.workspaces[workspaceLocation] as IWorkspaceState).globals
     );
+    const variant: Variant = yield select(
+      (state: IState) => (state.workspaces[workspaceLocation] as IWorkspaceState).context.variant
```

Comment:
We need to obtain the variant from the current context, whenever the editor's code is evaluated.